### PR TITLE
allow enable/disable for previous/next buttons.

### PR DESCRIPTION
**Pull request by joebnc**

### CODE REVIEW

File: src/material/paginator/paginator.ts [hunk 1]
```diff
@@ -82,6 +82,9 @@ export class MatPaginator extends _MatPaginatorBase implements OnInit, OnDestroy
   HasInitialized {
   private _initialized: boolean;
   private _intlChanges: Subscription;
+  
+  /**to allow overwrite the enable/disable tooltip for previous/next buttons*/
+  @input() previousNextButtonsTooltipDisabled:bool;
```
Comment (EdricChan03) [inline src/material/paginator/paginator.ts]: Couple of typos:

As TypeScript is case-sensitive, you have to use the capital form of `Input`.

There's also no such type as a `bool` in TypeScript: it should be `boolean`.

There should also be a whitespace in-between the variable and the type.

```suggestion
  @Input() previousNextButtonsTooltipDisabled: boolean;
```

File: src/material/paginator/paginator.ts [hunk 2]
```diff
@@ -82,6 +82,9 @@ export class MatPaginator extends _MatPaginatorBase implements OnInit, OnDestroy
   HasInitialized {
   private _initialized: boolean;
   private _intlChanges: Subscription;
+  
+  /**to allow overwrite the enable/disable tooltip for previous/next buttons*/
```
Comment (EdricChan03) [inline src/material/paginator/paginator.ts]: There should be whitespace in-between the start and end of the comment.

I'm also suggesting that the comment be renamed to this:

```suggestion
  /** Overrides the ability for the tooltip to be disabled for previous/next buttons */
```

**Comment by joebnc** [comment]
> Is there any reason for this change?
> 
> The tooltips should be disabled when the buttons are disabled as this is intentional behaviour.
> 
> (P.S. I'm not a collaborator of the repository and hence have no write rights to this PR.)

Agree on this behavior.
What I need is to have the ability to disable the tooltip without disable the buttons.

**Comment by EdricChan03** [comment]
@joebnc Why did you co-author me in your commits? I did not contribute to the code whatsoever.

**Comment by jelbourn** [comment]
Closing this since it seems stale and the cla is missing.

**Pull request closed without merging**